client/src/components/add-job-post-component/job-info-form/index.jsx

```diff
@@ -0,0 +1,243 @@
+            <MenuItem key={jobType} value={jobType}>
+              {jobType}
+            </MenuItem>
+          ))}
+        </Select>
+        <FormHelperText>*Required</FormHelperText>
+      </FormControl>
+
+      {/* Hours Per Week Field */}
+      <TextField
+        fullWidth
+        sx={{ m: 1, width: "47%" }}
+        id="hoursPerWeek"
+        type="number"
+        label="Hours per week"
+        inputProps={{ min: 0 }}
+        value={jobPostData.hoursPerWeek}
+        onChange={(e) => {
+          const { value } = e.target;
+          if (/^\d*\.?\d*$/.test(value))
+            handleInputChange(value, "hoursPerWeek");
+        }}
+        required
+        InputLabelProps={{ required: false }}
+      />
+
+      {/* Employment Type Field */}
+      <FormControl fullWidth sx={{ m: 1, width: "96%" }}>
+        <InputLabel>Employment Type</InputLabel>
+        <Select
+          sx={{ textAlign: "left" }}
+          labelId="employmentTypeLabel"
+          id="employmentTypeLabel"
+          value={jobPostData.employmentType}
+          label="Employment Type"
+          onChange={(e) => handleInputChange(e.target.value, "employmentType")}
+          required
+          InputLabelProps={{ required: false }}
+        >
+          {JOB_TYPES.map((jobType) => (
+            <MenuItem key={jobType} value={jobType}>
+              {jobType}
+            </MenuItem>
+          ))}
+        </Select>
+        <FormHelperText>*Required</FormHelperText>
+      </FormControl>
+
+      {/* Creation Date Picker */}
+      <LocalizationProvider dateAdapter={AdapterDayjs}>
+        <DatePicker
+          id="expirationDate"
+          label="Creation Date"
+          fullWidth
+          sx={{ m: 1, width: "47%" }}
+          value={jobPostData.creationDate}
+          onChange={(newValue) =>
+            handleInputChange(newValue, jobPostData, "creationDate")
+          }
+          minDate={dayjs()}
+          renderInput={(params) => (
+            // eslint-disable-next-line
+            <TextField {...params} error={false} helperText="" required />
+          )}
+          required
+          InputLabelProps={{ required: false }}
+        />
+      </LocalizationProvider>
+
+      {/* Expiration Date Picker */}
+      <LocalizationProvider dateAdapter={AdapterDayjs}>
+        <DatePicker
+          id="expirationDate"
+          label="Expiration Date"
+          fullWidth
+          sx={{ m: 1, width: "47%" }}
+          value={jobPostData.expirationDate}
+          minDate={dayjs()}
+          onChange={(newValue) =>
+            handleInputChange(newValue, jobPostData, "expirationDate")
+          }
+          renderInput={(params) => (
+            // eslint-disable-next-line
+            <TextField {...params} error={false} helperText="" required />
+          )}
+          required
+          InputLabelProps={{ required: false }}
+        />
+      </LocalizationProvider>
+
+      {/* Additional Information Field */}
```

Comment:
There might need to be additional fields for attributes such as close_date, job_description, and more. I might be wrong though!